pkg/csi_driver/node.go

```diff
@@ -292,7 +299,7 @@ func (s *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolu
 		}
 	}
 
-	if mounted && s.features.FeatureLockRelease {
+	if mounted && s.features.FeatureLockRelease.Enabled {
 		klog.V(4).Infof("NodeStageVolume mounted volume %v to staging target path %s, mount already exists. Proceed to lock info configmap updates", volumeID, stagingTargetPath)
 		if err := s.nodeStageVolumeUpdateLockInfo(ctx, req); err != nil {
 			return nil, status.Errorf(codes.Internal, "failed to store lock info after NodeStageVolume succeeded on volume %v to path %s: %v", volumeID, stagingTargetPath, err.Error())
```

Comment:
we can combine line if block of L302-309  and L310-L313
```
if mount {
   if feature enabled {
       nodeStageVolumeUpdateLockInfo()
       ...
   }
  klog.Info(NodeStageVolume suceeed ...)
}
```

Reply:
done